<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUniqueWindow</class>
 <widget class="QDialog" name="AddUniqueWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grabber - Ajouter une image</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Site</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Md5</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboSites"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineId"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineMd5"/>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Nom de fichier</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineFilename"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="labelFilename">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Dossier</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineFolder"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFolder">
       <property name="text">
        <string>Parcourir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddUniqueWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddUniqueWindow</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddUniqueWindow</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineId</sender>
   <signal>returnPressed()</signal>
   <receiver>AddUniqueWindow</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineMd5</sender>
   <signal>returnPressed()</signal>
   <receiver>AddUniqueWindow</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineFilename</sender>
   <signal>returnPressed()</signal>
   <receiver>AddUniqueWindow</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineFolder</sender>
   <signal>returnPressed()</signal>
   <receiver>AddUniqueWindow</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ok()</slot>
  <slot>add()</slot>
 </slots>
</ui>
